FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Multitask/demo_config.h

```diff
@@ -249,7 +249,7 @@ extern void vLoggingPrintf( const char * pcFormatString,
  * @brief Whether to use mutual authentication. If this macro is not set to 1
  * or not defined, then plaintext TCP will be used instead of TLS over TCP.
  */
-#define democonfigUSE_TLS                   1
+#define democonfigUSE_TLS                   0
```

Comment:
I thought we encouraged the use of TLS by default?

Reply:
I will change it back as the documentation says it is on by default.